internal/controllers/cryostat_controller_test.go

```diff
@@ -1278,6 +1270,27 @@ var _ = Describe("CryostatController", func() {
 				t.expectDeployment()
 			})
 		})
+		Context("with network options", func() {
+			JustBeforeEach(func() {
+				t.reconcileCryostatFully()
+			})
+			Context("containing core config", func() {
+				BeforeEach(func() {
+					t.objs = append(t.objs, test.NewCryostatWithCoreNetworkOptions())
+				})
+				It("should create the route as describe", func() {
```

Comment:
```suggestion
				It("should create the route as described", func() {
```

Reply:
Oh always have some typos :D that is funny haha Fixed it!